### ISSUE TITLE
About multi-label

### ISSUE THREAD

**Issue by josianerodrigues** [tengshaofeng/ResidualAttentionNetwork-pytorch#6]
Hi @tengshaofeng,
Do you know if this model can process multi-label datasets, like NUSWIDE? Any idea how to do it? Thank you.

**Comment by tengshaofeng** [comment]
multi-label means a image have several labels?  if yes, than you can modify the softmax in last layer to sigmoid,  each neuron is connected with sigmoid, and each neuron is responsible for one label.

**Comment by josianerodrigues** [comment]
Yes, multi-label means a image with more than one label. In that case, do you suggest any specific pytorch loss function?

**Comment by tengshaofeng** [comment]
nn.BCELoss
